hazelcast/src/main/resources/hazelcast-config-5.2.xsd

```diff
@@ -2534,10 +2534,11 @@
         </xs:simpleContent>
     </xs:complexType>
     <xs:complexType name="index">
-        <xs:all>
+        <xs:sequence>
```

Comment:
Isn't this breaking change? Before that something like 
```
<bitmap-index-options>....
<attributes>...
```
was legal, but now we enforce order of tags with xs:sequence

Reply:
True, reverted